### PR TITLE
Populate all the LFS data needed for running examples within the release container

### PR DESCRIPTION
**Pull request by AnuradhaKaruppiah**
Move the data files needed for log-parsing from models to examples/data to avoid copying models to the release container

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -360,7 +360,7 @@ COPY --from=git_clone "/tmp/morpheus_repo/conda/environments/*.yaml" "./conda/en
 COPY --from=git_clone "/tmp/morpheus_repo/docker" "./docker"
 COPY --from=build_docs "/workspace/${MORPHEUS_ROOT_HOST}/build/docs/html" "./docs"
 COPY --from=git_clone "/tmp/morpheus_repo/examples" "./examples"
-COPY --from=git_clone "/tmp/morpheus_repo/models/datasets" "./models/datasets"
+COPY --from=git_clone "/tmp/morpheus_repo/models" "./models"
```
Comment (mdemoret-nv) [inline docker/Dockerfile:363]: Remove this entire line and move the log-parsing and fraud detection files necessary to run examples to the `examples/data` folder

File: examples/data/log-parsing-config-20220418.json
Comment (dagardner-nv) [inline examples/data/log-parsing-config-20220418.json:1]: I think this should be a copy not a move

File: examples/data/log-parsing-validation-data-input.csv
Comment (dagardner-nv) [inline examples/data/log-parsing-validation-data-input.csv:1]: Same, copy

**Review by dagardner-nv — APPROVED** [review]
LGTM, but lets do copies and not moves.

**Pull request merged**